grammar.js

```diff
@@ -262,6 +265,28 @@ module.exports = grammar({
     keyword_cost: _ => make_keyword("cost"),
     keyword_rows: _ => make_keyword("rows"),
     keyword_support: _ => make_keyword("support"),
+    keyword_extension: _ => make_keyword("extension"),
+    keyword_out: _ => make_keyword("out"),
+    keyword_inout: _ => make_keyword("inout"),
+    keyword_variadic: _ => make_keyword("variadic"),
+
+    keyword_session: _ => make_keyword("session"),
+    keyword_isolation: _ => make_keyword("isolation"),
+    keyword_level: _ => make_keyword("level"),
+    keyword_serializable: _ => make_keyword("seria"),
+    keyword_repeatable: _ => make_keyword("repeatable: _ ="),
+    keyword_read: _ => make_keyword("read: _ =>"),
+    keyword_write: _ => make_keyword("wr"),
+    keyword_committed: _ => make_keyword("committed"),
+    keyword_uncommitted: _ => make_keyword("uncommitted"),
+    keyword_deferrable: _ => make_keyword("deferrable"),
+    keyword_names: _ => make_keyword("names"),
+    keyword_zone: _ => make_keyword("zone"),
+    keyword_immediate: _ => make_keyword("immediate"),
+    keyword_deferred: _ => make_keyword("deferred"),
+    keyword_constraints    : _ => make_keyword("constraints"),
+    keyword_snapshot: _ => make_keyword("snapshot"),
+    keyword_characteristics: _ => make_keyword("characteristics"),
```

Comment:
Most of these Keywords are not used at the moment, correct? I would prefer only adding those that are actually used in the parser.

Reply:
They are used in set_statement